js/exporter/exceljs/export_format.js

```diff
@@ -117,6 +117,39 @@ function _convertNumberFormat(format, precision, currency) {
     return result;
 }
 
+function _hasCSVInjection(value) {
```

Comment:
prefix underscore looks really strange for me)

Reply:
It is common styling for internal function in this file. It needs to refactor all functions and remove `_` for them. Such changes should be introduce in another PR.